src/promptflow-azure/promptflow/azure/_utils/_tracing.py

```diff
@@ -76,6 +78,8 @@ def validate_trace_destination(value: str) -> None:
         print(init_cosmos_msg)
         _logger.debug(init_cosmos_msg)
         pf_client._traces._setup_cosmos_db(resource_type=COSMOS_DB_SETUP_RESOURCE_TYPE)
```

Comment:
please update according to our latest discusssion

Reply:
updated